netci.groovy

```diff
@@ -73,27 +73,28 @@ class WcfUtilities
             return 
         }
 
-        wcfRepoSyncServiceCount++
+        wcfPRServiceCount++
```

Comment:
So in short, over the course of a typical day/week where no netci.groovy changes are being made, the wcfPRServiceId will remain the same.